addon/button.js

```diff
@@ -33,6 +34,53 @@ function initButton(sfHost, inInspector) {
     loadPopup();
   });
 
+
+  function addFlowScrollability() {
+    const currentUrl = window.location.href;
+    // Check the current URL for the string "builder_platform_interaction"
+    if (currentUrl.includes("builder_platform_interaction")) {
+      // Create a new checkbox element
+      const overflowCheckbox = document.createElement("input");
+      overflowCheckbox.type = "checkbox";
+      overflowCheckbox.id = "overflow-checkbox";
+      const checkboxState = localStorage.getItem("checkboxScrollState");
+      // Check local storage for the checkbox state
+      checkboxState ? overflowCheckbox.checked = JSON.parse(checkboxState) : overflowCheckbox.checked = true;   
+      // Create a new label element for the checkbox
+      const overflowLabel = document.createElement("label");
+      overflowLabel.textContent = "Enable flow scrollability";
+      overflowLabel.htmlFor = "overflow-checkbox";
+      if(currentUrl.includes("sandbox")){
```

Comment:
I understand that the UI is different from prod and sandboxes, have you tried to include the checkbox directly in the div with the class 'slds-builder-header flow-builder-header_border_icon' ? 
So that we don't have any difference from prod and sandbox plus the code is lighter 
What do you think about it ?
```suggestion
      if (currentUrl.includes("sandbox")){
```